### DISCUSSION TITLE
[RFC] Tag Performers and Studios, not just Scenes

### DISCUSSION THREAD

**Discussion by cinderblockgames** [stashapp/stash#5272] [Q&A]
I have my scenes under /Studios and /Models.  I can use the auto tagger to add a Studios or Models tag to Scenes, but what I really want is to be able to filter on the Studios and Performers pages down to just Studios that are under /Studios and Performers that are under /Models.  I'm not sure if this is currently doable.

Is there a way to either a) filter the Studios or Performers page based on Scenes under each item, or b) promote tags from Scenes to the Studios and Performers associated with them?  Or another way to accomplish this that I'm not seeing?

**Comment by feederbox826** [comment]
Yes, with https://github.com/stashapp/stash/pull/4962

Performers -> Filter -> Path -> `/Models`
Studio -> Filter -> Path -> `/Studios`

Adding tags from scenes to the performers will proably result in an explosion of bad tags, as scenes with two or more performers will see both inherting all tags (ie Average Male Body to female performers)

**Comment by cinderblockgames** [comment]
@feederbox826 Thank you!  Not published out yet, right?  But I will keep an eye out for it!
